subsys/bluetooth/host/classic/rfcomm.c

```diff
@@ -1748,7 +1748,7 @@ int bt_rfcomm_dlc_disconnect(struct bt_rfcomm_dlc *dlc)
 		 * and stop the tx thread.
 		 */
 		dlc->state = BT_RFCOMM_STATE_USER_DISCONNECT;
-		net_buf_put(&dlc->tx_queue,
+		k_fifo_put(&dlc->tx_queue,
```

Comment:
minor: there's a clang-format suggestion to put the call on a single line. Would also fix the misalignment of the split line that's caused by the new function being one character shorter.